src/JustSaying/Naming/IDefaultQueueNamingConvention.cs

```diff
@@ -0,0 +1,7 @@
+namespace JustSaying.Naming
+{
+    public interface IDefaultQueueNamingConvention
```

Comment:
Should this just be `IQueueNamingConvention`? (same for topic API)

Reply:
The reason for including default is so that it aligns with the comments on https://github.com/justeat/JustSaying/blob/master/src/JustSaying/Fluent/TopicSubscriptionBuilder%601.cs#L34 and https://github.com/justeat/JustSaying/blob/master/src/JustSaying/Fluent/QueueSubscriptionBuilder%601.cs#L34

Maybe the comments could be updated on the above to make it clear that these will be used?

Reply:
Ah, got it. The name makes sense (I was confused again 😄 )

Reply:
So this only kicks in where no string is passed in `ForTopic/ForQueue`

Reply:
Yeah, that's right. If you provide a string then that exact value is used